sentences/pl/_common.yaml

```diff
@@ -46,13 +46,19 @@ lists:
         out: "fahrenheit"
 expansion_rules:
   name: "{name}"
-  area: "[w] {area}"
-  what_is: "Jak[a] jest"
+  set: "(ustaw | zmień | przestaw)"
+  area: "[w|na] {area}"
+  what_is: "Jak[a|i] jest"
+  all: "(wszystkie | całe)"
   how_many_is: "Ile jest"
-  turn_on: "(włącz | zapal | uruchom)"
+  turn_on: "(włącz | zapal | oświeć | uruchom)"
```

Comment:
`turn_on` is a general extension used in all sort of situations and as we do not want to `oświeć` a ceiling fan a toaster, please remove it or move it outside of this extension.

Reply:
this problem was fixed